app/src/main/res/layout/activity_detail.xml

```diff
@@ -29,12 +38,28 @@
                 android:paddingTop="40dp">
 
                 <ImageView
+                    myImage="@{viewModel.movieLD.posterPath}"
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
                     android:layout_gravity="center"
                     android:adjustViewBounds="true"
-                    android:contentDescription="@string/content_description_movie_poster"
-                    app:imageUrl="@{movie.imageThumbnail}" />
+                    android:contentDescription="@string/content_description_movie_poster" />
+                <ProgressBar
+                    android:id="@+id/detail_activity_progress_barr"
+                    android:layout_width="64dp"
+                    android:layout_height="64dp"
+                    android:layout_gravity="center"
+                    android:visibility="@{viewModel.movieLD==null?View.VISIBLE:View.GONE}" />
```

Comment:
In viewModel you can create one line variable: 
boolean showProgressBar = movieLD==null
then here you can use it @{viewModel.showProgressBar ? View.VISIBLE : View.GONE}